build.sbt

```diff
@@ -90,3 +96,32 @@ lazy val mockExamples = crossProject(JSPlatform, JVMPlatform)
   .dependsOn(mock)
   .settings(stdSettings("zio-macros-mock-examples"))
   .settings(examplesSettings())
+
+lazy val delegate = crossProject(JSPlatform, JVMPlatform)
+  .in(file("delegate"))
+  .settings(stdSettings("zio-macros-delegate"))
+  .settings(macroSettings())
+  .settings(
+    scalacOptions --= Seq("-deprecation"),
+    scalacOptions ++= Seq("-Ywarn-unused:-patvars,-explicits,_")
+  )
+
+lazy val delegateTests = crossProject(JSPlatform, JVMPlatform)
+  .in(file("delegate-tests"))
+  .dependsOn(delegate)
+  .settings(stdSettings("zio-macros-delegate-tests"))
+  .settings(
+    skip in publish := true,
+    scalacOptions ++= Seq("-Ywarn-unused:-explicits,_")
+  )
+
+lazy val delegateExamples = crossProject(JSPlatform, JVMPlatform)
+  .in(file("delegate-examples"))
+  .dependsOn(delegate)
+  .settings(stdSettings("zio-macros-delegate-examples"))
+  .settings(
+    skip in publish := true,
+    libraryDependencies ++= Seq(
+      "dev.zio" %% "zio" % "1.0.0-RC11-1"
```

Comment:
Any reason not to use the latest `RC14`?

Reply:
Sure, will do